src/articles/RecommendedArticles.js

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import LoadingAnimation from "../components/LoadingAnimation";
+import { setTitle } from "../assorted/setTitle";
+import strings from "../i18n/definitions";
+
+import ArticlePreview from "./ArticlePreview";
+
+import SortingButtons from "./SortingButtons";
+
+import * as s from "../components/TopMessage.sc";
+
+export default function OwnArticles({ api }) {
+  const [articleList, setArticleList] = useState(null);
+
+  let originalList = articleList;
+
+  if (articleList == null) {
+    api.getRecommendedArticles((articles) => {
+        setArticleList(articles);
+    });
+  
+
+    setTitle("Recommended Articles");
+
+    return <LoadingAnimation />;
+  }
+
+  if (articleList.length === 0) {
```

Comment:
While this is an OK solution, it might feel strange for a user to have the tab, click it and then see no recommendations are available. Instead, what we could do is to check how many articles the user has liked and only show it once they have 5 (10?) whatever number you feel confident for the system to at least return some results.

Especially true for new users, who would see this tab and not know when the should revisit it.

Reply:
Added your suggestion of checking if the user has liked at least 5 articles before displaying the for you page. 

When testing locally, it displays the for you page after 5 is liked, if I unlike one of the articles, it disappears.

Would be much appreciated if you could test this on your local instance, to make sure you get the same results.